change/p-graph-2020-07-13-15-23-15-chrisgo-concurrency.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "minor",
```

Comment:
I think we should mark this as a major change: even if API is same, the implementation is so different that I believe warrants for a major bump.

Reply:
We're in 0.*, which has different semver implications. Do you want to bump to 1.0?

Reply:
Done